tools/mmp/driver.cs

```diff
@@ -358,8 +358,8 @@ static void Main2 (string [] args)
 					true /* this is an internal option */
 				},
 				{ "xamarin-framework-directory=", "The framework directory", v => { xm_framework_dir = v; }, true },
-				{ "xamarin-full-framework", "Used with --target-framework=4.5 to select XM 4.5 Target Framework", v => { IsUnifiedFullXamMacFramework = true; } },
-				{ "xamarin-system-framework", "Used with --target-framework=4.5 to select XM 4.5 Target Framework", v => { IsUnifiedFullSystemFramework = true; } },
+				{ "xamarin-full-framework", "Used with --target-framework=4.5 to select XM Full (4.5) Target Framework", v => { IsUnifiedFullXamMacFramework = true; } },
+				{ "xamarin-system-framework", "Used with --target-framework=4.5 to select XM Full (4.5) Target Framework", v => { IsUnifiedFullSystemFramework = true; } },
```

Comment:
I think we should remove the `(4.5)` from the descriptions, it's not correct and it's getting worse over time.